src/clj/harja/palvelin/komponentit/http_palvelin.clj

```diff
@@ -358,7 +369,7 @@
                                            :aktiiviset_pyynnot dec
                                            :pyyntoja_palveltu inc)))))
 
-                 {:port (or (:portti asetukset) asetukset)
+                 {:port portti
```

Comment:
Aah, edelliseen jatkona, olitkin vain kääntänyt tämän toisin päin, eli otetaan portti jo aiemmin letiin. Makes sense.

Reply:
:)